src/components/Icon/glyphs/index.js

```diff
@@ -34,6 +34,8 @@ import More from './More.svg';
 import MoreVertical from './MoreVertical.svg';
 import PaperClip from './PaperClip.svg';
 import Planet from './Planet.svg';
+import Plus from './_Plus.svg';
+import Salesforce from './Salesforce.svg';
```

Comment:
@ej9x we already have non-common icons directly in our app

Reply:
@zouxuoz  It is just a cloud. It could be used. I can rename it.

Reply:
I can name it like a Cloud

Reply:
@zouxuoz removed